backend/src/utils/componentUtils.ts

```diff
@@ -180,7 +180,8 @@ const getCSVForApp = (
 
   const installedCSV = subStatus.installedCSV;
 
-  if (!installedCSV) {
+  // eslint-disable-next-line
+  if (!!installedCSV) {
```

Comment:
any reason you couldn't just remove the `!` and not have to disable eslint?

Reply:
@gitdallas If `installedCSV` is present => the `fastify.kube.customObjectsApi.getNamespacedCustomObject` should execute successfully and not result in a `HTTP request failed`. I was trying to handle a case when this string is falsy... if it's falsy.. we should execute `Promise.resolve(undefined)` instead of `fastify.kube.customObjectsApi.getNamespacedCustomObject`

Reply:
`!!falsy === false` though, both `falsy` and `!!falsy` would evaluate to false here.

if you want "falsy to execute `Promise.resolve(undefined)`", the previous way was doing that. `!falsy === true`

Reply:
Agreed, this change would prevent `fastify.kube.customObjectsApi.getNamespacedCustomObject` from ever getting called with a defined `installedCSV` value, there must be something else going on to cause the bug